src/main/java/net/mcft/copy/backpacks/ProxyClient.java

```diff
@@ -88,8 +88,10 @@ public void init() {
 		
 		RenderManager manager = mc.getRenderManager();
 		Map<String, RenderPlayer> skinMap = manager.getSkinMap();
-		skinMap.get("default").addLayer(new RendererBackpack.Layer());
-		skinMap.get("slim").addLayer(new RendererBackpack.Layer());
+		for (RenderPlayer renderer : skinMap.values())
+		{
```

Comment:
Please match the coding style of the mod. We want to keep uniform codestyle.

Reply:
Ah, sorry. Yeah, I kept that in mind in the other files, but this was the first one I did, and I forgot to come back here.

Reply:
Should I change it, or will you just fix the style when potentially merging?

Reply:
You should change it. I have other issues to review.